menu.js

```diff
@@ -1,5 +1,35 @@
 const menu = {
-  // TODO
-};
-
-export default menu;
+    _courses : {
+      appetizers : [],
+      mains : [],
+      desserts : []
+      },
+      get courses() {
+        return this._courses;
+      },
+      addDishToCourse(courseName, dishName, dishPrice) {
+        let dish = {
+          name : dishName,
+          price : dishPrice
+        };
+        this._courses[courseName].push(dish);
+      },
+      getRandomDishFromCourse(courseName) {
+        let dishes = this._courses[courseName];
+        let randomIndex = Math.floor(Math.random() * dishes.length);
+        return dishes[randomIndex];
+      },
+      generateRandomMeal() {
+        let appetizer = this.getRandomDishFromCourse('appetizers');
+        let main = this.getRandomDishFromCourse('mains');
+        let dessert = this.getRandomDishFromCourse('desserts');
+        let dishes = [appetizer.name, main.name, dessert.name];
+        let total = appetizer.price + main.price + dessert.price;
```

Comment:
It seems we don't need these 2 variables as they are 
- used only once
- do not add great value to our code readability.